Update fusion readiness checklist for dbt jobs (#8377)
fishtown-analytics/docs.getdbt.com: website/docs/docs/fusion/fusion-readiness.md

We determine <Constant name="fusion" /> eligibility using data from your job runs. 

- [ ] Ensure you have at least one job running in each of your projects in the <Constant name="dbt_platform" />.

> - [ ] Ensure all jobs are running on the `Latest` release track.

Reviewer: ```suggestion
- [ ] Ensure all jobs are running on the [**Latest** release track](/docs/dbt-versions/cloud-release-tracks#which-release-tracks-are-available).
```

> - [ ] Resolve any job failures, jobs must be running successfully for eligibility checks to run. 

Reviewer: ```suggestion
- [ ] Resolve any job failures &mdash; all jobs must run successfully for eligibility checks to work.
```
